kiali/kiali
# Add Ambient label in service and app details

### Describe the change
Add Ambient label in service and app details

New labeling: 

![image](https://github.com/kiali/kiali/assets/49480155/1d1d1b9f-6f5a-476f-8e1d-c754090cb561)
![image](https://github.com/kiali/kiali/assets/49480155/2654baba-dc02-4fa2-8019-8c28be4134d4)

![image](https://github.com/kiali/kiali/assets/49480155/a5e8befc-2e93-4825-96cb-f35ab45d5f72)
![image](https://github.com/kiali/kiali/assets/49480155/ba98f7e2-b9f4-43fa-be3b-51cba0129f7f)

Included Label in Mesh page: 
![image](https://github.com/kiali/kiali/assets/49480155/33fa6eb6-5761-49fd-a862-87276c935406)


### Steps to test the PR

- Install istio ambient
- Add namespace to Ambient
- Check Services and App details, should include the Ambient Label

Note: Waypoint labels should be fixed here: https://github.com/kiali/kiali/pull/7351 so, L7 should not be visible for workloads

### Automation testing

N/A

### Issue reference

Fixes #7432
